src/sections/organizers.astro

```diff
@@ -3,17 +3,20 @@ import { getCollection } from "astro:content";
 import PersonCard from "../components/PersonCard.astro";
 
 const organizers = await getCollection("organizers");
+
+const listStyles = "flex flex-wrap justify-center gap-8 py-8 px-0";
+const listItemStyles = "list-item list-none pb-2";
```

Comment:
I suppose this utilities can be moved in class attribute, or if multiple components use them we can also create a custom "component" in ``tailwind.config.msj`